includes/CMB2.php

```diff
@@ -410,16 +412,16 @@ public function render_group( $args ) {
 
 		$field_group->peform_param_callback( 'before_group' );
 
-		echo '<div class="cmb-row cmb-repeat-group-wrap ', esc_attr( $field_group->row_classes() ), '" data-fieldtype="group"><div class="cmb-td"><div data-groupid="', esc_attr( $field_group->id() ), '" id="', esc_attr( $field_group->id() ), '_repeat" ', $this->group_wrap_attributes( $field_group ), '>';
+		echo '<div class="cmb-row cmb-repeat-group-wrap ', esc_attr( $field_group->row_classes() ), '" data-fieldtype="group"><div class="cmb-td"><div data-groupid="', esc_attr( $field_group->id() ), '" id="', esc_attr( $field_group->id() ), '_repeat" ', wp_kses( $this->group_wrap_attributes( $field_group ), $this->kses_args( 'group_wrap_attributes' ) ), '>';
```

Comment:
Basically, I'd rather know I can rely on the unit tests to tell me if something is going to break backwards compatibility than stare at it and guess (or test it manually myself). 

I know that it's a lot of assumptions, like _maybe_ someone decides to throw javascript in somewhere where it doesn't belong or whatever, but if that's a legitimate concern, let's test against it, so the tests can fail, we can figure out a sanitization solution that _works_ and then move on. And for some things I do think that a line in the sand needs to be drawn where the data makes sense.